app/models/github/team_member.rb

```diff
@@ -1,8 +1,5 @@
 class Github::TeamMember < ApplicationRecord
-  has_one :user,
-    foreign_key: :uid,
-    primary_key: :user_id,
-    class_name: "User",
-    inverse_of: :github_team_memberships,
-    dependent: :destroy
+  belongs_to :user,
+    primary_key: :uid,
+    inverse_of: :github_team_memberships
```

Comment:
I don't think this is right. Looking at the schema, there is a `user_id` column. So the primary key is the `user_id` column on this table. 

We can remove the user_id column, in which case we need to remove that column and add a `uid` column instead. But I feel like we should just be referencing a normal `user_id` instinctively.

Reply:
That would be better, as `user_id` is currently actually the `uid` (it's named user ID in github).

Reply:
The original idea was to decouple the table as we might want to query non-DB users, but that hasn't really happend. I'll do a separate PR to fix this. I'll also check the organization members.